src/stories/Button.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import './button.css';
+
+/** Primary UI component for user interaction */
+export const Button = ({ primary, backgroundColor, size, label, ...props }) => {
+  const mode = primary ? 'storybook-button--primary' : 'storybook-button--secondary';
+  return (
+    <button
+      type="button"
+      className={['storybook-button', `storybook-button--${size}`, mode].join(' ')}
+      style={backgroundColor && { backgroundColor }}
```

Comment:
Don't use inline styles for an elements.
You can use `variant` prop to set different types of classNames for styles

```jsx
<button variant="primary">Primary btn</button>
<button variant="secondary">Secondary btn</button>
<button variant="default">Default btn</button>
```

Reply:
that is the component added with storybook example files. I will not change them since it is not a part of main project.